### PR TITLE
Split errors

### PR DESCRIPTION
**Pull request by WyseNynja**
- [ ] 0 cost errors

### CODE REVIEW

File: entities/src/login.rs [hunk 1]
```diff
@@ -9,8 +8,7 @@ use serde::{Deserialize, Serialize};
 pub struct Model {
     #[sea_orm(primary_key)]
     pub id: u64,
-    #[sea_orm(unique)]
-    #[serde(serialize_with = "serialization::uuid_as_ulid")]
```
Comment (yenicelik) [inline entities/src/login.rs:13]: we don't need the serialize decorator anymore?

File: entities/src/pending_login.rs [hunk 2]
```diff
@@ -9,8 +8,7 @@ use serde::{Deserialize, Serialize};
 pub struct Model {
     #[sea_orm(primary_key)]
     pub id: u64,
-    #[sea_orm(unique)]
-    #[serde(serialize_with = "serialization::uuid_as_ulid")]
```
Comment (yenicelik) [inline entities/src/pending_login.rs:13]: same question as before

File: entities/src/revert_log.rs [hunk 3]
```diff
@@ -13,7 +12,7 @@ pub struct Model {
     pub rpc_key_id: u64,
     pub timestamp: DateTimeUtc,
     pub method: Method,
-    #[serde(serialize_with = "serialization::vec_as_address")]
```
Comment (yenicelik) [inline entities/src/revert_log.rs:16]: Same question as before. Also, is Vec<u8> intended? before, it was marked as a uuid

File: entities/src/rpc_key.rs [hunk 4]
```diff
@@ -10,8 +10,7 @@ pub struct Model {
     #[sea_orm(primary_key)]
     pub id: u64,
     pub user_id: u64,
-    #[sea_orm(unique)]
-    #[serde(serialize_with = "serialization::uuid_as_ulid")]
+    #[sea_orm(column_type = "Binary(BlobSize::Blob(Some(16)))", unique)]
```
Comment (yenicelik) [inline entities/src/rpc_key.rs:13]: same question as before

File: entities/src/user.rs [hunk 5]
```diff
@@ -9,8 +8,7 @@ use serde::{Deserialize, Serialize};
 pub struct Model {
     #[sea_orm(primary_key)]
     pub id: u64,
-    #[sea_orm(unique)]
-    #[serde(serialize_with = "serialization::vec_as_address")]
+    #[sea_orm(column_type = "Binary(BlobSize::Blob(Some(20)))", unique)]
```
Comment (yenicelik) [inline entities/src/user.rs:11]: again a Vec<u8>, no need to change with Address? also is the serialization decorator intended

File: web3_proxy/src/stats/influxdb_queries.rs [hunk 6]
```diff
@@ -575,7 +576,25 @@ pub async fn query_user_stats<'a>(
                             );
                         }
                         _ => {
-                            error!("error_response should always be a Long!");
+                            error!("error_response should always be a String!");
+                        }
+                    }
+                } else if stat_response_type == StatType::Detailed && key == "user_error_response" {
```
Comment (yenicelik) [inline web3_proxy/src/stats/influxdb_queries.rs:582]: just confirming that this was added to the influx schema

**Review by yenicelik — APPROVED** [review]
Looks good, a couple questions just to make sure

File: entities/src/login.rs (hunk 1 above)
Comment (WyseNynja) [reply entities/src/login.rs:13]: we do. just missed it being replaced

File: web3_proxy/src/stats/influxdb_queries.rs (hunk 6 above)
Comment (WyseNynja) [reply web3_proxy/src/stats/influxdb_queries.rs:582]: Don't we use a schemaless influx that gets set on first insert?

Comment (yenicelik) [reply web3_proxy/src/stats/influxdb_queries.rs:582]: i suppose we do

**Pull request merged**